### PR TITLE
[RDPHOEN-1240] Completing the "set-fix-refspecs" change with docu in …

### PR DESCRIPTION
**Pull request by jan-hannig**
…README

### CODE REVIEW

**Comment by jan-hannig** [comment]
Gitlab Pipeline running here:
https://gitlab.devops.defra01.iris-sensing.net/MIRROR/iris-kas/-/pipelines/1542

**Comment by jan-hannig** [comment]
Please find and read the corresponding manual in our confluence:

File: README.md [hunk 1]
```diff
@@ -152,6 +153,20 @@ Commands:
 - `[ENV_VARS] make force-pull`
 
 
+### Set a firmware tag
```
Comment (Jasper-Ben) [inline README.md:156]: Please change to: `Version pinning for thirdparty layer repositories`

File: README.md [hunk 2]
```diff
@@ -152,6 +153,20 @@ Commands:
...
+
+*Will set and commit fixed refspecs in the thirdparty layer repos file (kas-irma6-base-common.yml) for "tagging" a firmware containing a not committed branch*
```
Comment (Jasper-Ben) [inline README.md:158]: Please change to:

Will automatically set fixed commit hashes for thirdparty layer repos in `kas-irma6-base-common.yml`. Useful for temporary version pinning of the base software

File: README.md [hunk 3]
```diff
@@ -17,6 +17,7 @@
     - [Run interactive QEMU VM](#run-interactive-qemu-vm)
     - [Update all repos](#update-all-repos)
     - [Force update all repos](#force-update-all-repos)
+    - [Set a firmware tag](#set-a-firmware-tag)
```
Comment (Jasper-Ben) [inline README.md:20]: Please change to: `Version pinning for thirdparty layer repositories`

**Review by Jasper-Ben — CHANGES_REQUESTED** [review]

**Review by Jasper-Ben — APPROVED** [review]

**Pull request merged**